pkg/schedule/schedulers/hot_region.go

```diff
@@ -152,6 +176,22 @@ func (h *baseHotScheduler) prepareForBalance(rw statistics.RWType, cluster sche.
 			prepare(regionWrite, constant.LeaderKind)
 			prepare(regionWrite, constant.RegionKind)
 			h.updateWriteTime = time.Now()
+			// calculate the probability to move peer or transfer leader
+			total, count := 0.0, 0.0
+			for _, store := range h.stLoadInfos[writePeer] {
+				for _, peer := range store.HotPeers {
+					total += float64(len(peer.GetStores()))
```

Comment:
Do you needs to filter some unhealty peer like pending peer or down peer?

Reply:
added `IsRegionReplicated`